components/butterfly/README.md

```diff
@@ -0,0 +1,62 @@
+# Butterfly
+
+Buttefly is the Habitat gossip protocol. It's an implementation of
```

Comment:
As it turns out, I live in Butte county, and I can confirm that the flies here are lovely.